### PR TITLE
Landing Page and redirect on login

### PR DESCRIPTION
**Pull request by SMoore3773**
Routed to landing page on main page load.
Reroute to homepage on successful login.
[Trello Card](https://trello.com/c/v7RN8vIp)

Line 45 of index.js for the main app has a linting error because of the way that okta is storing it's token.

Changed title of login screen to reflect City Spire app

### CODE REVIEW

File: src/components/pages/Landing/RenderLandingPage.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 
 function RenderLandingPage(props) {
+
+  const history = useHistory();
+
+  const loginHandler = () =>{
+    history.push('/login');
+  };
+
+
```
Comment (allyjay317) [inline src/components/pages/Landing/RenderLandingPage.js:14]: Is this intended to be reusable or have anything added to it later? or would it be good to do inline on line 27 instead?

**Review by allyjay317 — CHANGES_REQUESTED** [review]
Generally looks good, possibly move the homepagePushHandler to a useEffect instead?

**Review by allyjay317 — APPROVED** [review]
Looking good!

**Review by songamugenzi — APPROVED** [review]

**Comment by allyjay317** [comment]
[Create Landing Page](https://trello.com/c/Zq1DwK4V/36-create-landing-page)

**Comment by allyjay317** [comment]
[Create unprotected landing page](https://trello.com/c/v7RN8vIp/34-create-unprotected-landing-page)

**Pull request merged**